docs/getting-started.md

```diff
@@ -80,99 +84,285 @@ await Kilt.init({ address: YOUR_CHAIN_ADDRESS })
-      type: 'string',
-    },
-    age: {
-      type: 'integer',
-    },
-  },
-  type: 'object',
+const keystore = new Kilt.Did.DemoKeystore()
+
+// Signing keypair
+const claimerSigningKeypair = await keystore.generateKeypair({
+  alg: Kilt.Did.SigningAlgorithms.Ed25519,
+  seed: generateClaimerMnemonic,
+})
+
+// Encryption keypair
+const claimerEncryptionKeypair = await keystore.generateKeypair({
+  alg: Kilt.Did.EncryptionAlgorithms.NaclBox,
+  seed: generateClaimerMnemonic,
 })
 ```
 
-### 3.2. Storing a CTYPE
+### 2.2. Generate a light DID for the Claimer
 
-Before you can store the CTYPE on the blockchain, you have to configure your blockchain address and connect to it.
-Either explicitly:
+Using the keys from the demo keystore to generate the claimers light DID.
 
 ```typescript
-Kilt.config({ address: YOUR_CHAIN_ADDRESS })
-await Kilt.connect()
+import Kilt from '@kiltprotocol/sdk-js'
+
+const claimerLightDid = new Kilt.Did.LightDidDetails({
+  authenticationKey: {
+    publicKey: claimerSigningKeypair.publicKey,
+    type: Kilt.Did.DemoKeystore.getKeypairTypeForAlg(claimerSigningKeypair.alg),
+  },
+  encryptionKey: {
+    publicKey: claimerEncryptionKeypair.publicKey,
+    type: Kilt.Did.DemoKeystore.getKeypairTypeForAlg(
+      claimerEncryptionKeypair.alg
+    ),
+  },
+})
+
+// DID light: `did:kilt:light:014qFxmHnWw5sGMwjskdvMCrASF9Jvu5ggWRTWTK2NNYSLDg56:oWFlomlwdWJsaWNLZXlYIJuIow7rjSdf92qMKYtWV42lF9mctD1nFf8RM24auJhwZHR5cGVmeDI1NTE5`
+console.log(claimerLightDid.did)
 ```
 
-Or by setting a default address in the configuration, connecting implicitly.
+### 2.3 Generate a full DID for the Attester
+
+Before we can send the request for an attestation to an Attester, we should first create an Attester identity and a full on-chain DID. Using the previously generated keyring.
 
 ```typescript
-Kilt.config({ address: YOUR_CHAIN_ADDRESS })
+import Kilt from '@kiltprotocol/sdk-js'
+import { mnemonicGenerate } from '@polkadot/util-crypto'
+
+// mnemonic: coast ugly state lunch repeat step armed goose together pottery bind mention
+const attesterMnemonic = mnemonicGenerate()
+
+const attester = keyring.createFromUri(attesterMnemonic)
+
+console.log('attester address', attester.address)
 ```
 
-Note that calling (as described in [1.2](#initializing-the-kilt-sdk))
+Providing the attester with funds from the endowed account in order to write transactions on-chain.
 
 ```typescript
-Kilt.init({ address: YOUR_CHAIN_ADDRESS })
+import Kilt from '@kiltprotocol/sdk-js'
+
+const transferAmount = '1000000000000000'
+await Kilt.Balance.makeTransfer(attester.address, transferAmount).then((tx) =>
+  Kilt.BlockchainUtils.signAndSubmitTx(tx, identity, {
+    resolveOn: Kilt.BlockchainUtils.IS_FINALIZED,
+    reSign: true,
+  })
+)
 ```
 
-initializes the SDK _and_ sets the config, so it is related to the second approach.
+Then we generate all the keypairs for the attester to construct the attestation, delegation and authentication keys with the keystore.
```

Comment:
We should specify that we use the same key for authentication, attestation, and delegation but that this is not recommended for production use.

Reply:
I have added a note.